src/services/socket.js

```diff
@@ -0,0 +1,38 @@
+import io from 'socket.io-client';
+import { dispatch } from '../store/store';
+import { setGameState, setMessage } from '../store/actions';
+
+const connectSocket = () => {
+    let connectionString;
+    if (process.env.NODE_ENV === "development") {
+        connectionString = "ws://localhost:8080";
+    } else {
+        connectionString = `wss://${window.location.host}`;
```

Comment:
This approach will work for now but we might want to look into a more robust/cleaner approach.